.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
 before_install:
   - npm install -g npm@latest
+install:
+  - npm install --global electron
```

Comment:
I thought that Electron tended to be running on a slightly different version to "mainstream" NodeJS?
(and I guess we should probably target the same version of Electron in here as Etcher is using?)